www/include/configuration/configObject/host_template_model/hostTemplateModel.php

```diff
@@ -37,17 +37,18 @@
     exit();
 }
 
-isset($_GET["host_id"]) ? $hG = $_GET["host_id"] : $hG = null;
-isset($_POST["host_id"]) ? $hP = $_POST["host_id"] : $hP = null;
-$hG ? $host_id = $hG : $host_id = $hP;
-
-isset($_GET["select"]) ? $cG = $_GET["select"] : $cG = null;
-isset($_POST["select"]) ? $cP = $_POST["select"] : $cP = null;
-$cG ? $select = $cG : $select = $cP;
-
-isset($_GET["dupNbr"]) ? $cG = $_GET["dupNbr"] : $cG = null;
-isset($_POST["dupNbr"]) ? $cP = $_POST["dupNbr"] : $cP = null;
-$cG ? $dupNbr = $cG : $dupNbr = $cP;
+$host_id = filter_var(
+    call_user_func(function () {
+        if (isset($_GET["host_id"])) {
```

Comment:
$host_id -> $hostId

what would happen if $GET['host_id'] is a string, (which will return $host_id = false) ?

Reply:
Read the function 'getDuplicateNumberOption'